polaris-icons/svgo.config.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
```

Comment:
Pulled from https://github.com/Shopify/polaris/blob/main/polaris-icons/rollup.config.mjs#L109

Can they be shared?